src/test/local/codecept.conf.js

```diff
@@ -0,0 +1,60 @@
+const browserstack = require("browserstack-local");
+require('dotenv').config()
+
+const BROWSERSTACK_USERNAME = process.env.BROWSERSTACK_USERNAME
+const BROWSERSTACK_ACCESS_KEY = process.env.BROWSERSTACK_ACCESS_KEY
+
+exports.config = {
+    tests: './*_test.js',
+    output: './output',
+    helpers: {
+        WebDriver: {
+            url: 'https://bstackdemo.com',
+            user: BROWSERSTACK_USERNAME,
+            key: BROWSERSTACK_ACCESS_KEY,
+            browser: 'chrome',
+            capabilities: {
+                "os": "Windows",
+                "os_version": "10",
+                "browser_version": "latest",
+                "browserstack.local": "true",
+                "project": "Codecept + WebdriverIO",
+                "build": "Local_Execution",
+                "name": "Local Test",
+            }
+        }
+    },
+
+    bootstrap: function () {
+        console.log("Connecting Local");
+        exports.bs_local = new browserstack.Local();
+        exports.bs_local.start({ 'key': BROWSERSTACK_ACCESSKEY }, function (error) {
```

Comment:
Currently it's not waiting for local spawn. Let's create a promise for bs_local.start method and await in bootstrap async function.